frontend/src/Schedulr.js

```diff
@@ -48,6 +60,56 @@ class Schedulr extends React.Component {
     this.setState({...this.state, scheduleList: listOfSchedules});
   }
 
+  generateSchedulesTimes() {
+    this.state.scheduleList.forEach((schedule) => {
+      this.generateScheduleMeetingTimes(schedule);
+    });
+    return true;
```

Comment:
You should consider adding an eslint rule for line breaks around code blocks

https://eslint.org/docs/rules/padding-line-between-statements

```
"padding-line-between-statements": [
  { "blankLine": "always", "prev": "block-like", "next": "*" },
  { "blankLine": "always", "prev": "*", "next": "block-like" }
]
```

I think that rule might work. Haven't tested it though, so YYMV but the point is to enforce new lines before and after code blocks, eg.

```
function foo() {
  const a = 2;
  if (a > 3) {
    console.log('oh no');
  }
  return 'apple';
}
```

Would become

```
function foo() {
  const a = 2;

  if (a > 3) {
    console.log('oh no');
  }

  return 'apple';
}
```

This could be done in a short follow-up PR

Reply:
Sounds good, we will add that, thank you!